src/commands.rs

```diff
@@ -320,6 +320,10 @@ pub struct ElectrumOpts {
         default_value = "10"
     )]
     pub stop_gap: usize,
+
+    /// Enable domain validation when connecting to Electrum servers.
+    #[clap(name = "VALIDATE_DOMAIN", long = "validate_domain")]
```

Comment:
In the `utils.rs` the default for this value is true, as you have it here the user will be required to set the value to `true` or `false`. It's better if you set the clap `default_value` to true then the user only needs to do something if they want it to be false.

Reply:
Done. Also added a `v` short option.